code/__HELPERS/text.dm

```diff
@@ -53,6 +53,36 @@
 /proc/sanitize(t,list/repl_chars = null)
 	return html_encode(sanitize_simple(t,repl_chars))
 
+//Used for preprocessing entered text
+/proc/sanitize_rp(var/input, var/max_length = MAX_MESSAGE_LEN, var/encode = 1, var/trim = 1, var/extra = 1)
```

Comment:
I'd rename this to sanitize_multiline, and specifically annotate in a comment that, unlike sanitize, this does **not** strip newlines